src/main/java/com/sleekbyte/tailor/format/Formatter.java

```diff
@@ -1,17 +1,37 @@
 package com.sleekbyte.tailor.format;
 
+import com.sleekbyte.tailor.common.ColorSettings;
+import com.sleekbyte.tailor.common.ExitCode;
 import com.sleekbyte.tailor.output.ViolationMessage;
 
+import java.io.File;
 import java.io.IOException;
 import java.util.List;
 
 /**
- * Interface used by the Printer class to display violation messages/errors.
- * New formats can be added by implementing this interface.
+ * Formatter used by the Printer class to display violation messages.
```

Comment:
The formatter also gets used in Tailor main, so can we remove `used by the Printer class` since it implies that these Formatter methods will only be called by Printer.